src/locales/en.json

```diff
@@ -0,0 +1,69 @@
+{
+    "translation": {
```

Comment:
Don't need `translation` key

Reply:
fix: translation key removed in both en & uk files